### ISSUE TITLE
"Start File" ignored under some conditions.

### ISSUE THREAD

**Issue by xopxe** [pkulchenko/ZeroBraneStudio#1070]
When the edited file is Lua and there is a "Start File" set, then the Start File is correctly run when hitting F6/F5 . If the current tab is not a .lua file but another extension added to luaspec.exts, then will try to execute it ignoring the Stat File. 

In my case, these are conf files I added to my user setting with:
```lua
luaspec.exts[#luaspec.exts+1] = "conf"
```

**Comment by pkulchenko** [comment]
@xopxe, I can't reproduce this issue; I always get the debugging started on the correct file. Is there anything else of interest? Can you remove all the plugins and see if the issue still happens?

**Comment by xopxe** [comment]
Yes, removed all the plugins and it sill happens for me. But there are some observations:

* It only happens with the default "Lua" selected as the interpreter in the menu. If I switch to another interpreter (Lua5.3 in my case) everything works correctly.
* When I right click on the file in the side bar and select "Set as Start File", only has effect when the file is also the current tab (this might be related to the other strange menu behavior).

**Comment by pkulchenko** [comment]
> When I right click on the file in the side bar and select "Set as Start File", only has effect when the file is also the current tab (this might be related to the other strange menu behavior).

I suspect it is related to the same issue as discussed in #1062, as all these menu items are using `GetFocusedItem` call, which doesn't seem to be working correctly when the local menu is opened.

**Comment by xopxe** [comment]
Yes, I guess it's unrelated to what I see here.

I just confirmed that when add an extension to `luaspec.exts`, these files when active will ignore a Start File and try to execute.
